keps/sig-network/2449-move-externalDNS-out-of-kubernetes-incubator/kep.yaml

```diff
@@ -0,0 +1,5 @@
+title: Move ExternalDNS out of Kubernetes incubator
+kep-number: 2449
+authors:
+  - "@njuettner"
+owning-sig: sig-network
```

Comment:
@shekhar-rajak - this failed validation because of lack of "status: implemented"

please fix

Reply:
Thanks for review I will do this

Reply:
done